src/resources/locations.js

```diff
@@ -0,0 +1,5 @@
+import createResourceDescription from './util/createResourceDescription'
+
+export const locations = createResourceDescription('locations', {
```

Comment:
Is this correct? I don’t think we have a root locations endpoint? I think it will either be a subresource of appointment or an include on appointment?